### DISCUSSION TITLE
Pseudo-organizational changes in ASA Stats :)

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#101] [Announcements]
As you may already know, our [whitepaper](https://www.asastats.com/whitepaper.pdf) brought some (at that time) functional titles prefixed with "First".

Based on that document we have got First Developer, First Admin, First Debater, First Contributor, First Writer, and First Investor. By having those titles, those people had earned the possibility to buy 5 presale slots instead of the regular 2 defined for the rest of the Asastatser group.

Meanwhile, in [December](https://www.asastats.com/transparency-report-2021-12.pdf) the new members of the community have been selected as the First Investor and the First Admin, and the additional 3 slots are offered to purchase in an out-of-presale arrangement to each of them. Among other reasons for that change, we wanted to reward those two prominent members for their exceptional contribution to the project. 

And so on that December we have sort of defined that holding that title represents having an active position in the project. This announcement is on the same track like that.

First Developer title has been introduced for the founder of the project to show on his own example his unconditional trust in the future of the project. Now it's time for a community member to take that title.

First Debater has been a valuable and trustworthy community member the whole time. Still, this is [by the definition](https://github.com/asastats/channel/wiki/Discussions) the most important role in the ASA Stats community and it implies time spent and dedication to the project.

ASA Stats project owns a lot to First Writer, probably only one other community member can compare to that. First Writer will be rewarded with the most valuable ASA Stats NFT once we get it, but the time is for another community member to take the title.

As the old benefits of having the extra presale slots are obsoleted now, so we have to announce a new way of rewarding the title holders: the new titleholders will get additional 618,034 governor seat tokens in the very next update.

First Developer - @dragmz 
First Debater - @SCN9A 
First Writer - @AlgoRhythMatic 

Congratulations!

**Comment by Babbexx22** [comment]
Congratulations. The titles were all earned, and well deserved. ASA Stats is proud to have y'all around

**Reply by ipaleka** [reply]
Yep, and afaic the only thing that's left is to invent a title for you mate...

**Reply by Motuwagon** [reply]
First facilitator, for babex they always help facilitate the project fulfilling needs as conversations or projects demand.

**Reply by ipaleka** [reply]
> First facilitator, for babex they always help facilitate the project fulfilling needs as conversations or projects demand.

That describes him well, but IMO we should instead go with a function that demands the title, rather than another way around.

For example, once we start with designing and conducting our marketing campaign, there could be a position and a person behind that position who knows what has been decided and what should be done. If Babbexx would have got enough time to be around we can go with First Advertiser. :)

**Comment by AlgoRhythMatic** [comment]
Thank you!  It is always a great pleasure to work on this project w/ everyone in this community!

**Comment by lario0913** [comment]
Congratulations mates...

Lario; first 🤔 🤔
